Tests/SentryTests/Networking/RateLimits/SentryConcurrentRateLimitsDictionary.swift

```diff
@@ -65,10 +73,26 @@ class SentryConcurrentRateLimitsDictionaryTests: XCTestCase {
         
         for i in Array(0...10) {
             let date = self.currentDateProvider.date().addingTimeInterval(TimeInterval(i))
-            XCTAssertEqual(date, sut.getRateLimit(forCategory: "A\(i)"))
-            XCTAssertEqual(date, sut.getRateLimit(forCategory: "B\(i)"))
-            XCTAssertEqual(date, sut.getRateLimit(forCategory: "C\(i)"))
-            XCTAssertEqual(date, sut.getRateLimit(forCategory: "D\(i)"))
+            
+            let a = i as NSNumber
+            let b = 100 + i as NSNumber
+            let c = 200 + i as NSNumber
+            let d = 300 + i as NSNumber
+            
+            XCTAssertEqual(date, sut.getRateLimit(for: getCat(rawValue: a)))
+            XCTAssertEqual(date, sut.getRateLimit(for: getCat(rawValue: b)))
+            XCTAssertEqual(date, sut.getRateLimit(for: getCat(rawValue: c)))
+            XCTAssertEqual(date, sut.getRateLimit(for: getCat(rawValue: d)))
         }
     }
+    
+    private func getCat(rawValue:NSNumber) -> SentryRateLimitCategory {
```

Comment:
Can we name like this?

```suggestion
    private func getCategory(rawValue:NSNumber) -> SentryRateLimitCategory {
```